tuf/developer_tool.py

```diff
@@ -391,6 +391,7 @@ def status(self):
         self._log_status(self.project_name, signable, self.repository_name)
 
       except securesystemslib.exceptions.Error as e:
+        # pylint: disable=unsubscriptable-object
```

Comment:
OT: This seems to be the reason why the current [TUF@develop build is failing](https://travis-ci.org/theupdateframework/tuf). IMO it should be fixed in a separate PR. Such a fix should probably also evaluate, whether pylint has a point here.

Reply:
https://github.com/theupdateframework/tuf/pull/839 resolves this issue.  (Sorry, folks.)